LICENSE

```diff
@@ -1,21 +1,8 @@
-MIT License
```

Comment:
Not sure why the license was altered...

Reply:
By mistake I guess, reverting it right now